docs/source/get_started/custom_ui.rst

```diff
@@ -0,0 +1,105 @@
+=========
+Custom UI
+=========
+
+.. contents:: Table of Contents
+
+------------
+Introduction
+------------
+
+Custom UI is a feature for enhancing default behavior of the embedded grids that originate from the MDS module. It
+allows high customization of those grids by overriding the default implementation of the javascript methods.
+
+-----------
+Preparation
+-----------
+
+There are few steps you need to take in order to be able to use the Custom UI feature. First of all, you're going to
+need a controller that will handle the :code:`/mds-databrowser-config` requests sent to your module. This controller
+should return a string containing the overriding javascript methods that will be used in place of the default MDS ones.
+
+Here's a code example:
```

Comment:
From what I understand this can also be static file instead of method that uses the config. Maybe mention that.